File: Programming Fundamentals/C#/04.Methods/Lab/P02_Grades/P02_Grades.cs
namespace P02_Grades
{
    using System;

    class P02_Grades
    {
        static void Main(string[] args)
        {
            double grade = double.Parse(Console.ReadLine());

            PrintGradeInWords(grade);
        }

        public static void PrintGradeInWords(double grade)
        {

            if (grade >= 2.00 && grade < 3.00)
            {
                Console.WriteLine("Fail");
            }
            else if (grade >= 3.00 && grade < 3.50)
            {
                Console.WriteLine("Poor");
            }
            else if (grade >= 3.50 && grade < 4.50)
            {
                Console.WriteLine("Good");
            }
            else if (grade >= 4.50 && grade < 5.50)
            {
                Console.WriteLine("Very good");
            }
            else if (grade >= 5.50 && grade <= 6.00)
            {
                Console.WriteLine("Excellent");
            }
        }
    }
}

File: Programming Fundamentals/JS/04.Functions/Lab/p02_Grades.js
function grade(grade) {
  if (grade >= 2.00 && grade <= 2.99) {
    return 'Fail';
  } else if (grade >= 3.00 && grade <= 3.49) {
    return 'Poor';
  } else if (grade >= 3.50 && grade <= 4.49) {
    return 'Good';
  } else if (grade >= 4.50 && grade <= 5.49) {
    return 'Very good';
  } else if (grade >= 5.50 && grade <= 6.00) {
    return 'Excellent';
  }
}

// Don't copy the calling of the function in judge, you won't get any points, just the code above
console.log(grade(3.33));
console.log(grade(4.50));
console.log(grade(2.99));
